msteams-platform/sbs-meetings-stage-view.yml

```diff
@@ -14,102 +14,108 @@ metadata:
+    Share meeting stage in Teams helps to enable and configure your apps for Teams meetings. It helps users to share meeting stage, allows meeting participants to add status details of past, present, and future tasks, enhancing Teams meeting experience.
 
-    **Key features of Meetings Stage View**
+    **Key features of share meeting stage in Teams**
 
-    * Add status details in side panel.
-    * Add details for collaboration.
+    * Add status details in the side panel
+    * Add details for collaboration
     
-    This step-by-step guide helps you to generate Meetings Stage View to add status details for collaboration in Teams. You'll see the following output:
+    This step-by-step guide helps you to share meeting stage in Teams to add status details for collaboration. You'll see the following output:
 
-      :::image type="content" source="../msteams-platform/assets/images/sbs-meetings-stage-view/stage view.png" alt-text="meetings stage view" border="false":::
+      :::image type="content" source="../msteams-platform/assets/images/sbs-meetings-stage-view/stage view.png" alt-text="meetings stage view" border="true":::
 
 - title: Prerequisites
   durationInMinutes: 1
   content: |
-    Ensure you install the following tools and set up your development environment:  
 
-    * [Microsoft Teams with an account (not a guest account)](https://developer.microsoft.com/microsoft-365/dev-program)
-    * [Visual Studio code](https://code.visualstudio.com/Download)
-    * [nodejs](https://nodejs.org/en/download/) LTS version 10.21.0 or later
-    * [ngrok](https://ngrok.com/download) latest version (only for devbox testing) or any equivalent tunneling solution such as Command Prompt
+    Ensure you install the following tools and set up your development environment:
+
+    * [Microsoft Teams](https://teams.microsoft.com/) with valid account
+    * [Latest version of Visual Studio](https://visualstudio.microsoft.com/downloads/)
+    * [Latest version of nodejs](https://nodejs.org/en/download/) 
+    * [Latest version of ngrok](https://ngrok.com/download) (only for devbox testing) or any equivalent tunneling solution
     
       > [!NOTE]
       > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
 
-    Ensure you have working knowledge of the following technologies:
+     You must have working knowledge of the following technologies:
 
     * [C#](https://docs.microsoft.com/dotnet/csharp/fundamentals/tutorials/how-to-display-command-line-arguments)
     * [ECMAScript6](http://es6-features.org/#Constants)
-    * [ASP.NET core](https://docs.microsoft.com/aspnet/core/?view=aspnetcore-3.1) version 3.1
-    * [React.JS](https://reactjs.org/tutorial/tutorial.html) version 16 or later
+    * [Latest version of ASP.NET core](https://docs.microsoft.com/aspnet/core/?view=aspnetcore-3.1) 
```

Comment:
3.1 is not the latest version of Dotnet. Also, the latest version of dotnet does not have the 'core' suffix

Reply:
Resolved